src/sagemaker/pytorch/model.py

```diff
@@ -137,34 +136,21 @@ def prepare_container_def(self, instance_type, accelerator_type=None):
                 For example, 'ml.p2.xlarge'.
             accelerator_type (str): The Elastic Inference accelerator type to
                 deploy to the instance for loading and making inferences to the
-                model. For example, 'ml.eia1.medium'.
+                model. Currently unsupported with PyTorch.
```

Comment:
Should we instead remove this param entirely? Alternatively, add a warning to spew this out if it's passed in?

Reply:
a warning does get printed out, but the logic is in a different spot

code: https://github.com/aws/sagemaker-python-sdk/blob/master/src/sagemaker/fw_utils.py#L56, https://github.com/aws/sagemaker-python-sdk/blob/master/src/sagemaker/fw_utils.py#L273-L277
unit test: https://github.com/aws/sagemaker-python-sdk/blob/master/tests/unit/test_pytorch.py#L346-L352